LambdaEngine/Source/Rendering/RenderGraph.cpp

```diff
@@ -3291,6 +3303,19 @@ namespace LambdaEngine
 		return true;
 	}
 
+	bool RenderGraph::CustomRenderStagesPostInit()
+	{
+		for (auto& customRenderer : m_CustomRenderers)
+		{
+			if (!customRenderer->RenderGraphPostInit())
```

Comment:
```suggestion
		for (auto pCustomRenderer : m_CustomRenderers)
		{
			if (!pCustomRenderer->RenderGraphPostInit())
```
Since these are pointers, reference is not necessary.